setup-production.adoc

```diff
@@ -31,217 +31,172 @@ $ cd taiga-docker/
-----
+Please, do not modify it unless you know what you’re doing.
 
-**Service: taiga-events**
-[source, bash]
-----
-RABBITMQ_USER: taiga
-RABBITMQ_PASS: taiga
-----
+.**Public registration**
+If you want to allow a public register, configure this variable to "True". By default it's "False". The value should be the same in `taiga-front` and `taiga-back`.
 
-**Service: taiga-async-rabbitmq**
+Service: `taiga-back`
 [source, bash]
 ----
-RABBITMQ_ERLANG_COOKIE: secret-erlang-cookie
-RABBITMQ_DEFAULT_USER: taiga
-RABBITMQ_DEFAULT_PASS: taiga
-RABBITMQ_DEFAULT_VHOST: taiga
+PUBLIC_REGISTER_ENABLED: "True"
 ----
 
-**Service: taiga-events-rabbitmq**
+Service: `taiga-front`
 [source, bash]
 ----
-RABBITMQ_ERLANG_COOKIE: secret-erlang-cookie
-RABBITMQ_DEFAULT_USER: taiga
-RABBITMQ_DEFAULT_PASS: taiga
-RABBITMQ_DEFAULT_VHOST: taiga
+PUBLIC_REGISTER_ENABLED: "true"
 ----
 
-.Taiga protected settings
+.**Session cookies in Django Admin**
 
-**Service: taiga-protected**
+Taiga doesn't use session cookies in its API as it stateless. However, the Django Admin (`/admin/`) uses session cookie for authentication. By default, Taiga is configured to work behind HTTPS. If you're using HTTP (despite the strong recommendations against it), you'll need to configure the following environment variables so you can access the Admin:
+
+Service: `taiga-back`
 [source, bash]
 ----
-MAX_AGE: 360
+SESSION_COOKIE_SECURE: "False"
+CSRF_COOKIE_SECURE: "False"
 ----
 
-The attachments will be accesible with a token during MAX_AGE (in seconds). After that, the token will expire.
-
-=== Customisation
+More info about those variables can be found link:https://docs.djangoproject.com/en/3.1/ref/settings/#csrf-cookie-secure[here].
 
-All these features are disabled by default. You should add the corresponding environment variables with a proper value to enable them.
 
-.Registration Settings
+.**Events and asynchronous tasks**
 
-**Service: taiga-back**
-[source, bash]
-----
-PUBLIC_REGISTER_ENABLED: "True"
-----
+Both events and asynchronous tasks in Taiga use RabbitMQ as their message queue system. There are two variables to set their connection URLs: `CELERY_BROKER_URL` (asynchronous tasks in taiga-back) and `EVENTS_PUSH_BACKEND_URL` (taiga-front events).
 
-**Service: taiga-front**
+Service: `taiga-back`
 [source, bash]
 ----
-PUBLIC_REGISTER_ENABLED: "true"
+EVENTS_PUSH_BACKEND_URL: "amqp://taiga:taiga@taiga-events-rabbitmq:5672/taiga"
+CELERY_BROKER_URL: "amqp://taiga:taiga@taiga-async-rabbitmq:5672/taiga"
 ----
 
-If you want to allow a public register, configure this variable to "True". By default is "False". Should be the same as this var in `taiga-front` and `taiga-back`.
-
 **Important**: Taiga (in its default configuration) disables both Gitlab or Github oauth buttons whenever the public registration option hasn't been activated. To be able to use Github/ Gitlab login/registration, make sure you have public registration activated on your Taiga instance.
```

Comment:
this "important" was related to "public registration" (before the changes) and now is kind of hanging nowhere.

Reply:
Mea culpa. I'll change its order to be at the end of "public registration". I moved here the Django admin section not being aware of this part. Thanks